Makefile

```diff
@@ -0,0 +1,25 @@
+docstyle:
+	echo "Checking documentation with pydocstyle..."
+	python -m pydocstyle ${SRC_DIR}/pymc4/
+	echo "Success!"
+
+
+format:
+	echo "Checking code style with black..."
+	python -m black -l 100 --check pymc4/
+	echo "Success!"
+
+
+style:
+	echo "Checking code style with pylint..."
+	python -m pylint pymc4/
+	echo "Success!"
+
+
+test:
+	pytest -v pymc4/tests/ --cov=pymc4/
```

Comment:
Would be nice to add html output coverage output here for non computer users

Reply:
Do you have any opinions/prior knowledge on the best directory to store the output? 

We can generate a "testing-report.html", and add it to gitignore so it doesn't accidentally get checked into source control, perhaps.

Reply:
The output isn't just one html file, I think its a whole directory of html files. Adding it to `.gitignore`. is a great idea

Reply:
We're good - I figured out how to make it self-contained, with no assets directory necessary.